modules/@apostrophecms/schema/index.js

```diff
@@ -519,8 +519,30 @@ module.exports = {
       name: 'date',
       vueComponent: 'AposInputString',
       convert: async function (req, field, data, object) {
+        if (!data[field.name] && object[field.name]) {
+          // Allow date fields to be unset.
+          object[field.name] = null;
+          return;
+        }
+        if (field.min && dayjs(data[field.name]).isBefore(field.min)) {
```

Comment:
The nice thing about YYYY-MM-DD strings is that you can compare them directly, there's no need for dayjs for this purpose.

Reply:
This is true, but unless we enforce `YYYY-MM-DD` for `min/max` values, which we could do, we'd need the help here.

Reply:
We should enforce that for min/max I think. We don't want to turn dayjs into our official standard of what inputs are allowed, because then we're on dayjs forever. YYYY-MM-DD is what we store and what we allow for configuration, although we might allow other things in the UI.